cmd/new.go

```diff
@@ -57,6 +58,7 @@ func init() {
 	draftCmd.Flags().StringVar(&draftOpts.Package, "package", "", "package identifier for the release")
 	draftCmd.Flags().BoolVar(&draftOpts.NoAutoUpgrade, "no-auto-upgrade", false, "disable automatic upgrade checks [$KEYGEN_NO_AUTO_UPGRADE=1]")
 	draftCmd.Flags().StringSliceVar(&draftOpts.Entitlements, "entitlements", []string{}, "comma seperated list of entitlement constraints (e.g. --entitlements <id>,<id>,...)")
+	draftCmd.Flags().StringToStringVar(&draftOpts.Metadata, "metadata", map[string]string{}, "metadata for the release as key-value pairs (e.g. --metadata key1=value1,key2=value2)")
```

Comment:
This is not what I was thinking. This property should accept a JSON object as a string, not a map of strings — the map of strings doesn't support nested objects.